debian/changelog

```diff
@@ -1,3 +1,27 @@
+ostree (2017.7-1-endless1) unstable; urgency=medium
```

Comment:
I don't think this version is what you want. Having multiple `-` will make the upstream version `2017.7-1`.
```
$ python3
>>> from debian.debian_support import Version
>>> v = Version('2017.7-1-endless1')
>>> v.upstream_version
'2017.7-1'
>>> v.debian_revision
'endless1'
```
Normally there's no hyphen between the debian revision and the downstream version. E.g., `1endless1`.

This also could cause issues with the version calculation in `jenkins-obs-build`. It expects (incorrectly) that you'll start the debian revision at 0 and any subsequent commits on the debian branch will increase that. However, since the upstream branch has many commits after the upstream tag, the upstream version will be newer than what you have here and it won't matter.

So, I think you can leave this as `2017.7-1endless1` since we know it won't be an issue here. Eventually I will make it so that the versions can be more flexible...